config.py

```diff
@@ -7,6 +7,8 @@
 
 mongoEngineConfig = {
     'db': 'roomie',
-    'host': os.getenv('DATABASE_HOST'),
-    'port': os.getenv('DATATBASE_PORT')
+    'host':'localhost',
+    'port': 27017
+    # 'host': os.getenv('DATABASE_HOST'),
+    # 'port': os.getenv('DATATBASE_PORT')
```

Comment:
Instead of doing this, create an .env file in your local hand set ie DATABASE_HOST=localhost